src/model_exploration.py

```diff
@@ -0,0 +1,40 @@
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import classification_report
+from sentiment_exploration import *
```

Comment:
it's better to import specifically what you need
try something like:
`from sentiment_exploration import apply_sentiment_analysis`